templates/web/base/alert/_list.html

```diff
@@ -6,88 +6,106 @@
-        <div>
-    <h3>[% loc('Problems in an area') %]</h3>
-  [% END %]
+    <div class="govuk-radios" data-module="govuk-radios">
 
-  <p id="rss_local">
-    <label class="label-containing-checkbox" for="[% rss_feed_id %]">
-      <input type="radio" name="feed" id="[% rss_feed_id %]" value="[% rss_feed_id %]"[% IF rss_feed_id == selected_feed || selected_feed == '' %] checked[% END %]>
-     [% IF c.cobrand.moniker == 'cheshireeast' %]
-      [% tprintf( 'Reported problems within %.1fkm of %s', population_radius, name_of_location ) %]
-     [% ELSE %]
-      [% tprintf( loc('Problems within %.1fkm of %s'), population_radius, name_of_location ) %]
-     [% END %]
-      [% loc('(covers roughly 200,000 people)') %]
-    </label>
-  </p>
+      [% IF reported_to_options %]
+        <p>
+            [% tprintf(loc('Reports near %s are sent to different councils, depending on the type of problem.'), name_of_location) %]
+        </p>
+        <p>
+            [% loc('You can choose to subscribe to all problems reported in an area, or reports based on their destination.') %]
+        </p>
+        <div class="alerts__columns">
+            <div>
+        <h3>[% loc('Problems in an area') %]</h3>
+      [% END %]
 
-  <p id="rss_local_alt">
-    <label class="inline" for="distance">[% loc('Or specify a different radius distance:') %]</label>
-    <input class="form-control" name="distance" id="distance" type="text" inputmode="numeric" pattern="[0-9]*" aria-label="[% loc('Specify a radius in kilometers') %]"> km
-    <a href="[% rss_feed_uri %]" class="alerts-rss-link js-alert-local" id="rss-[% rss_feed_id %]">
-      <img src="/i/feed.png" width="16" height="16" title="[% loc('RSS feed of nearby problems') %]" alt="[% loc('RSS feed') %]">
-    </a>
-  </p>
+      
+        <div class="govuk-radios__item" id="rss_local">
+          <input class="govuk-radios__input" type="radio" name="feed" id="[% rss_feed_id %]" value="[% rss_feed_id %]"[% IF rss_feed_id == selected_feed || selected_feed == '' %] checked[% END %]>
+          <label class="govuk-label govuk-radios__label" for="[% rss_feed_id %]">
+          [% IF c.cobrand.moniker == 'cheshireeast' %]
+            [% tprintf( 'Reported problems within %.1fkm of %s', population_radius, name_of_location ) %]
+          [% ELSE %]
+            [% tprintf( loc('Problems within %.1fkm of %s'), population_radius, name_of_location ) %]
+          [% END %]
+            [% loc('(covers roughly 200,000 people)') %]
+          </label>
+        </div>
+
+        <div class="alert-option-wrapper govuk-!-margin-bottom-4">
+          <div id="rss_local_alt" class="govuk-form-group">
+            <label class="govuk-label govuk-!-margin-bottom-2" for="distance">[% loc('Or specify a different radius distance:') %]</label>
+            <div class="govuk-input__wrapper">
+              <input class="govuk-input" name="distance" id="distance" type="text" inputmode="numeric" pattern="[0-9]*">
```

Comment:
I think this has lost some changes you've made on master since starting this e.g. the aria-label?